gamemodes/sss/core/server/init.pwn

```diff
@@ -999,6 +999,7 @@ public OnScriptInit()
 
 
 	// SETTING HOLSTERABLE ITEMS
+	SetItemTypeHolsterable(item_Spade,			1, 0.179000, -0.032000, 0.000000, 0.000000, -112.199981, -78.399993, 600, "GOGGLES",	"GOGGLES_PUT_ON");
```

Comment:
Why would you want to holster a spade?

Reply:
I didn't realize that not even one of the melee weapons was set for this type, I'm used play on the GM that I coded for my server, but melee weapons, in general, could be put in the holster, it would give a certain "agility" for a pvp.

For example, on my server, players used to knockout others with the gun, and kill them with the knife, sword, pade, baton etc. and with it in the holster the process became faster and more effective.

Just an idea, I can commit with all melee weapons defined as Holsterable, my server was like that, what do you think about? I believe that it is not something that will "hinder" the gameplay, I think that it would just add a new usability, because it would not block the item to being stored in the inventory.

Reply:
Okay, this makes more sense from a gameplay perspective now. I understand what you mean. I originally enabled certain weapons to be holstered for this reason such as the knife, baton and sword.

The reason I haven't added _all_ melee weapons to be holsterable is partially balancing (so there are a set of rarer items that facilitate this convenience) and partially a pseudo-realism design decision (you can imagine holstering a knife in your jeans or a sword on your back ninja-style, or a baton in your belt-hole - but a spade is an odd choice, it's not something you can imagine as realistic - not that realism is a primary driver for all design decisions, it's just a factor for making the user-interface predictable for players.)

Reply:
The balancing factor is more of a reason I wouldn't add _all_ melee items. Doing so would mean that there are fewer incentives to seek out better melee items. If you can find a spade within 5 minutes of spawning and it's viable for use forever, there's little incentive to search for and pick up a slightly better melee item because you can just holster the spade as your off-hand.

However, if the spade (and similar items) were left as a hold-only melee item, it serves its purpose as a "last resort" weapon to grab from the environment that you'd then discard after a small fight in favour of a slightly better (or even worse) item that _can_ be holstered.

The main focus here is variety. Providing players with a set of items with different characteristics. Maybe the spade does more blunt damage (knockout chance) and _can't_ be holstered, but a weapon like the baton does _less_ damage but _can_ be holstered. This provides players with a set of tradeoffs to make a decision based on their own personal playstyle. If all melee items could be holstered, then there would be absolutely no reason to ever use the baton because it does less damage and has no advantage over the spade.

Reply:
Oh, I just noticed the issue you referenced! Sorry. I understand where the confusion comes from now.

I suppose this was an old joke of ours (Sheriffic liked to use them!) however, since we don't really play any more and there are plenty more little references in the game for us.